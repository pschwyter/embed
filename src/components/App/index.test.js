/* eslint-env node, jest */

import { h } from "preact";
import render from "preact-render-to-string";

describe("Hello logic", () => {
  it("should be able to run tests", () => {
    expect(1 + 2).toEqual(3);
  });
});


