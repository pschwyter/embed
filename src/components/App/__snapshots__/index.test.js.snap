// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should match snapshot 1`] = `
preact-render-spy (1 nodes)
-------
<div
  id="ada-embed"
  class="ada-chaperone-app"
>
  <div class="ada-chaperone-drawer ada-chaperone-drawer--hidden">
    <div
      class="ada-chaperone-drawer__mask"
      onClick={[Function bound ]}
      role="button"
    >
    </div>
    <div
      class="ada-chaperone-drawer__iframe-container"
      role="dialog"
      aria-modal={false}
      aria-hidden={true}
    >
    </div>
  </div>
  <div class="ada-chaperone-button-container">
    <button
      title="Open Support Chat"
      accessKey="1"
      class="ada-chaperone-button"
      onClick={[Function bound ]}
    >
      <undefined class="ada-chaperone-button__icon"></undefined>
    </button>
  </div>
</div>

`;
